integration-test/src/main/java/com/scalar/db/api/DistributedStorageAdminRepairIntegrationTestBase.java

```diff
@@ -57,14 +57,23 @@ public abstract class DistributedStorageAdminRepairTableIntegrationTestBase {
 
   protected DistributedStorageAdmin admin;
 
-  protected AdminTestUtils adminTestUtils;
+  protected AdminTestUtils adminTestUtils = null;
```

Comment:
Do we really need this change?

Reply:
I had to set it to null expressly otherwise Spotbug raises this warning since it believes this field is not being initialized
https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#uwf-unwritten-public-or-protected-field-uwf-unwritten-public-or-protected-field

Reply:
Oh okay. I understand. Thanks.